ndk/src/signers/private-key/index.ts

```diff
@@ -1,15 +1,15 @@
 import type { UnsignedEvent } from "nostr-tools";
-import { generatePrivateKey, getPublicKey, getSignature, nip04 } from "nostr-tools";
+import { generateSecretKey, getPublicKey, finalizeEvent, nip04 } from "nostr-tools";
 
 import type { NostrEvent } from "../../events/index.js";
 import { NDKUser } from "../../user";
 import type { NDKSigner } from "../index.js";
 
 export class NDKPrivateKeySigner implements NDKSigner {
     private _user: NDKUser | undefined;
-    privateKey?: string;
+    privateKey?: Uint8Array;
```

Comment:
This was one of the biggest changes in nostr-tools v2. Private keys are now always `Uint8Array`